hello_vsi/README.md

```diff
@@ -1,32 +1,68 @@
 
 ## Build and run
 
-Use the cbuild tool or an IDE to build the application project in csolution format (See the project [README](../README.md).
+Use the cbuild tool or an IDE to build the application project in csolution format (see the project [README](../README.md)).
 
-All applications have been ported for Corstone-300, Corstone-310 and Corstone-315, and can be compiled with Arm Compiler 6 or GCC. By default the Arm Compiler 6 is used, to compile with GCC, use `--toolchain GCC` option for cbuild command.
+All applications have been ported for **Corstone-300**, **Corstone-310** and **Corstone-315**, and can be compiled with **Arm Compiler** 6 or **GCC**. By default the Arm Compiler 6 is used, to compile with GCC, use `--toolchain GCC` option for the cbuild command.
 
 
 ### Application Without GUI
 
 Streams data from the `intdata.txt` file and prints to the console. For each loop, the application will wait until it get an interrupt from the driver that a new item has been streamed, at that point the application will proceed to copy the data from a data buffer and print out to the console.
 
-Use `.event` as the build type, followed by the target platform. For example, to build and run on Corstone-310, use:
+Use `.event` as the build type, followed by the target platform. For example:
 
-Compile: 
+To build the **Corstone-310** target with **GCC** use: 
 
 ```bash
-cbuild hello_vsi.csolution.yml -c .event+Corstone_310 --packs --update-rte --toolchain GCC
+cbuild hello_vsi.csolution.yml --context-set --update-rte --packs --context hello_vsi.event+Corstone_310 --toolchain GCC
+```
+
+To build the **Corstone-300** target with **AC6** (default toolchain): 
+
+short form
+```bash
+cbuild hello_vsi.csolution.yml --update-rte --packs --context hello_vsi.event+Corstone_300
+```
+
+or more specific
+
+```bash
+cbuild hello_vsi.csolution.yml --update-rte --packs --context hello_vsi.event+Corstone_300 --toolchain AC6
+```
+
+To build the **Corstone-315** target with **AC6**:
+```bash
+cbuild hello_vsi.csolution.yml --update-rte --packs --context hello_vsi.event+Corstone_315
 ```
 
-Run:
+
+To run a specific context:
+
+target **Corstone_310** which has been build with **GCC**  for **60 sec** use:
+
+```bash
+FVP_Corstone_SSE-310 -a ./out/hello_vsi/Corstone_310/event/hello_vsi.elf -C mps3_board.v_path=./source/VSI/data_sensor/python/ --simlimit 60
+```
+
+
+target **Corstone_300** which has been build with **AC6**  for **60 sec** use:
 
 ```bash
-FVP_Corstone_SSE-310 -a ./out/hello_vsi/Corstone_310/event/hello_vsi.elf -C mps3_board.v_path=./source/VSI/data_sensor/python/
+FVP_Corstone_SSE-300 -a ./out/hello_vsi/Corstone_300/event/hello_vsi.axf -C mps3_board.v_path=./source/VSI/data_sensor/python/ --simlimit 60
 ```
 
+
+target **Corstone_315** which has been build with **AC6**  for **60 sec** use:
```

Comment:
No need to duplicate, it is clear that different model and firmware shall be used for different target.